### ISSUE TITLE
Add tooltip to the top right buttons

### ISSUE THREAD

**Issue by mmazzarolo** [mmazzarolo/chrome-another-tab#1]
Easy one, add a tooltip to the: 
- Show/hide hidden folders button
- Change theme button

**Comment by ngensimore** [comment]
Hi I'll do this if that's cool.

**Comment by ONeill45** [comment]
@mmazzarolo Saw this was still open so went ahead and jumped on it, happy to push it up and open a PR.

**Comment by mmazzarolo** [comment]
@ONeill45 yes, absolutely, thanks! 👍 Please point to the `develop` branch. 
Let me know if you need any help.

**Comment by ONeill45** [comment]
@mmazzarolo branched off of `develop` and it's giving me permission denied to push back up to the repo

**Comment by mmazzarolo** [comment]
You mean you can submit the PR?

**Comment by ONeill45** [comment]
It won't let me push my branch up so I can create the PR

**Comment by dkozickis** [comment]
@ONeill45 you have to fork out to your own GitHub repo and then open PR from there :)

**Comment by ONeill45** [comment]
@mmazzarolo ahh gotcha, used to opening a new branch not forking. I'll get the PR in today
